Part 4 - Platform Features/README.md

```diff
@@ -25,8 +25,8 @@ We can easily check to see if our user is connected to the internet with the bui
 
     ```csharp
     builder.Services.AddSingleton<IConnectivity>(Connectivity.Current);
-    builder.Services.AddSingleton<IGeolocation>(Geolocation.Default);
-    builder.Services.AddSingleton<IMap>(Map.Default);
+    builder.Services.AddSingleton<IGeolocation>(Geolocation.Current);
+    builder.Services.AddSingleton<IMap>(Map.Current);
```

Comment:
They are both .Default for me..  https://github.com/dotnet/maui/blob/main/src/Essentials/src/Geolocation/Geolocation.shared.cs#L37

Current is private